### DISCUSSION TITLE
Pull Shark X3

### DISCUSSION THREAD

**Discussion by kirill-katkov** [Schweinepriester/github-profile-achievements#644] [Q&A]
GitHub Profile Achievements
https://github.com/Schweinepriester/github-profile-achievements
Hello! 128 pull requests merged but I never got the badge x3.
Tell me what is the reason?
Thank you in advance

<img width="676" alt="Снимок экрана 2023-02-17 в 13 47 32" src="https://user-images.githubusercontent.com/95480051/219624392-a1f7d505-3435-471b-9e51-bb7457232068.png">

**Comment by onwp** [comment]
Terms for badges might be updated.

**Reply by kirill-katkov** [reply]
but there is no information about this. I think the problem is different. I hope to get an answer

**Comment by 0xabdulkhaliq** [comment]
## PullShark Badge Requirements
- X1 (**TIER 1**) => 2 PRs
- X2 (**TIER 2**) => 16 PRs
- X3  (**TIER 3**) => 128 PRs
- X4  (**TIER 4**) => 1024 PRs
- If you done Pull Request in a right way then it must be accounted, if not make sure to done it right

**Comment by port19x** [comment]
This screenshot merely says you have 128 PRs closed. This does not take into account whether they were actually merged or rejected and closed.

**Comment by fournarin** [comment]


**Reply by port19x** [reply]
<img width="474" height="414" alt="image" src="https://github.com/user-attachments/assets/d020c1a1-932e-427c-848f-de0ec3bdbbbc" />
